manager/volume.go

```diff
@@ -190,6 +190,18 @@ func (m *VolumeManager) Create(name string, spec *types.VolumeSpec) (v *longhorn
 		}
 		logrus.Infof("Use the default number of replicas %v", spec.NumberOfReplicas)
 	}
+
+	if string(spec.DataLocality) == "" {
+		defaultDataLocalityMode, err := m.GetSettingValueExisted(types.SettingNameDefaultDataLocality)
+		if err != nil {
+			return nil, errors.Wrapf(err, "cannot get valid mode for setting default data locality mode for volume: %v", name)
+		}
+		spec.DataLocality = types.DataLocality(defaultDataLocalityMode)
+	}
+	if err := types.ValidateDataLocalityMode(spec.DataLocality); err != nil {
```

Comment:
Need to be consistent with the name, e.g. DataLocality instead of DataLocalityMode.

`Locality` is a noun so you don't need to mention `mode` every time.

Reply:
Thank you! Fixed